fetch.bs

```diff
@@ -1084,10 +1085,10 @@ Unless stated otherwise, it is "<code>basic</code>".
 <hr>
 
 <p>A <dfn export>subresource request</dfn> is a <a for=/>request</a>
-whose <a for=request>destination</a> is "<code>font</code>",
-"<code>image</code>", "<code>manifest</code>", "<code>media</code>",
-"<code>script</code>", "<code>style</code>", "<code>xslt</code>", or the empty
-string.
+whose <a for=request>destination</a> is "<code>fetch</code>",
+"<code>font</code>", "<code>image</code>", "<code>manifest</code>",
+"<code>media</code>", "<code>script</code>", "<code>style</code>",
+"<code>xslt</code>", or the empty string.
```

Comment:
If we're special-casing preload behavior here, what's the functional difference from second point we proposed in https://github.com/w3c/preload/issues/80#issuecomment-268573775?  We can just add a clause in Preload that maps "fetch" -> "".